lib/node_modules/@stdlib/math/base/tools/hermitepoly/README.md

```diff
@@ -0,0 +1,144 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Physicist's Hermite Polynomial
+
+> Evaluate a physicist's [Hermite polynomial][hermite-polynomial].
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+The physicist's [Hermite polynomials][hermite-polynomial] are given by
+
+<!-- <equation class="equation" label="eq:physicists_hermite_polynomials" align="center" raw="H_{n}(x)=(-1)^{n} e^{x^2} \frac{\mathrm d^{n}}{\mathrm d x^{n}} e^{-x^2}" alt="Equation for physicist's Hermite polynomials."> -->
```

Comment:
@jrreed83 I believe the `raw` equation in the HTML comment is missing the `{}` after both `\mathrm` calls.